plugins/linux/vm.sh

```diff
@@ -4,18 +4,18 @@ print_vm() {
     printf "%s\`%s\tL\t%s\n" $1 $2 $3
 }
 
-MEM=($(free -b | grep ^Mem:))
+MEM=($(free -k | grep ^Mem:))
 MEM_TOTAL=${MEM[1]}
 # For consistency across platforms, count cache as free, not used
 let MEM_USED=${MEM[1]}-${MEM[3]}-${MEM[5]}-${MEM[6]}
 let MEM_FREE=${MEM[3]}+${MEM[5]}+${MEM[6]}
-MEM_PERC=`echo "scale=2; $MEM_USED/$MEM_FREE" | bc`
+let MEM_PERC=$MEM_USED/$MEM_TOTAL
```

Comment:
This results in a consistent 0.00 in my short test running this on a CentOS7 VM. Since bc is not guaranteed to be available. Maybe awk would be a better alternative to get a floating point number.

MEM_PERC=$(awk -v u=$MEM_USED -v t=$MEM_TOTAL 'BEGIN { print u / t }')

The the MEM_USED and MEM_FREE calculations still result in bad output for CentOS7 specifically. MEM[6] is "available" memory in CentOS7's free. Both MEM[3] and MEM[6] are representing slightly different interpretations of memory available for applications. With the existing calculations the used result is negative and the free result is always greater than what is physically on the host.